app/assets/stylesheets/generic/select.scss

```diff
@@ -0,0 +1,4 @@
+.select {
+  padding: 2px 5px 2px 5px;
```

Comment:
Properties should be ordered height, padding<br>Shorthand form for property `padding` should be written more concisely as `2px 5px` instead of `2px 5px 2px 5px`